storage/storageUnit/storageunit.go

```diff
@@ -112,6 +115,14 @@ func (s *Unit) Put(key, data []byte) error {
 	return err
 }
 
+// Close will close unit
+func (s *Unit) Close() error {
+	err := s.persister.Close()
+	log.LogIfError(err)
```

Comment:
unless there is something different you are logging compared to what you are returning, I wouldn't do both.

Reply:
fixed